### PR TITLE
periphery 3.0.0

### PR DESCRIPTION
**Pull request by ileitch**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ileitch looks like it failed to detect the brew-installed swift?

```
  ==> /opt/homebrew/Cellar/periphery/3.0.0/bin/periphery scan --strict --skip-build --json-package-manifest-path manifest.json
  error: This version of Periphery only supports Swift >= 5.10, you're using Apple Swift version 5.9.2 (swiftlang-5.9.2.2.56 clang-1500.1.0.2.5)
```

File: Formula/p/periphery.rb [hunk 1]
```diff
@@ -15,6 +15,8 @@ class Periphery < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "fdfc5440f8eac20b8290a4173cadfc81090b12ef78454bed09a1a5ef56e30b54"
   end
 
+  depends_on xcode: ["15.4", :build, :test]
+
```
Comment (chenrui333) [inline Formula/p/periphery.rb:19]: with this, then that would affectively drop the ventura support?

Comment (chenrui333) [reply Formula/p/periphery.rb:19]: for the ventura test, you can try with

```
  if MacOS.version <= :ventura
    ENV.prepend_path "PATH", Formula["swift"].opt_bin
  end
```

Comment (ileitch) [reply Formula/p/periphery.rb]: > with this, then that would affectively drop the ventura support?

Yes. I've added `depends_on macos: :sonoma`.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**